libs/datamodel/core/src/transform/ast_to_dml/lift.rs

```diff
@@ -380,6 +380,31 @@ impl<'a> LiftAstToDml<'a> {
         } else if UNSUPPORTED_REGEX.is_match(type_name) {
             let captures = UNSUPPORTED_REGEX.captures(type_name).unwrap();
             let type_definition = captures.get(1).expect("get type definition").as_str();
+            if let Some(source) = self.source {
+                if captures.name("suffix").is_none() {
```

Comment:
I remember why it is, but I think a short comment here would help us remember why the absence of suffix is the criterion here

Reply:
done